spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -759,6 +759,51 @@ public void testPartitionedImportFromEmptyPartitionDoesNotThrow() {
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
+  @Test
+  public void testSkipOnError() throws IOException {
```

Comment:
I would probably split this into two tests one for it skipping on error, and one fore it not skipping on error. We should also add one for an invalid partition format if possible although i'm not sure we can manage that easily.

Reply:
Actually, looking at this again. I think this test with both the Skip off and ON is fine. We really don't have a better way of checking for invalid files